LibreHardwareMonitorLib/Hardware/Cpu/IntelCpu.cs

```diff
@@ -429,6 +431,15 @@ public IntelCpu(int processorIndex, CpuId[][] cpuId, ISettings settings) : base(
                 ActivateSensor(_coreVoltage);
             }
 
+            _coreVIDs = new Sensor[_coreCount];
+            coreSensorId = 1;
```

Comment:
This assignment seems pointless (also on L439).

Reply:
Oh, I forgot to remove those lines! I was using them on another test. Fixed.